cool.ldr.lfy.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "cool.ldr.lfy",
```

Comment:
I don't see how `ldr.cool` relate to this.

Is this really under the control of the application developers?

Reply:
sorry, I'm a newcomer, I don't understand what's wrong with `ldr.cool`

Reply:
> I don't see how `ldr.cool` relate to this.
> 
> Is this really under the control of the application developers?

Oh, I understand, the domain name ldr.cool now belongs to me

Reply:
> the domain name ldr.cool now belongs to me

It seems to be for an unrelated android app https://www.coolapk.com/apk/com.lzu.yuh.lzu If it belongs to you please add something about this app on the webpage.

Reply:
> > the domain name ldr.cool now belongs to me
> 
> It seems to be for an unrelated android app https://www.coolapk.com/apk/com.lzu.yuh.lzu If it belongs to you please add something about this app on the webpage.

Can I create a new second-level domain name lfy.ldr.cool or second-level directory ldr.cool/lfy? introduce this software here. The content on the ldr.cool website has other uses at present. Maybe I can introduce this software specifically on ldr.cool in the future.

Reply:
Both works I think.

Reply:
> Both works I think.

the website is ok: https://ldr.cool/lfy/

Reply:
Thanks!